rmgpy/cantherm/molpro.py

```diff
@@ -195,6 +193,16 @@ def loadConformer(self, symmetry=None, spinMultiplicity=None, opticalIsomers=1,
         line = f.readline()
         while line != '':
 
+            # Read the spin multiplicity if not explicitly given
+            if spinMultiplicity == 0 and 'spin' in line:
```

Comment:
I couldn't find the spin multiplicity anywhere in a molpro log other that the input echo, e.g.:
```
 basis=6-311G(2d,d,p)
 int;
 wf,spin=0,charge=0;
 {uks,B3LYP}
```
which means that users who use the alternative format:
```
 basis=6-311G(2d,d,p)
 int;
 wf,33,1,1;
 {uks,B3LYP}
```
or anything else, would have to manually specify the multiplicity.
Let me know if you have a different recommendation for solving this.

Reply:
Ah, I see. Yeah, I usually don't specify spin in the input file because I mostly just have stable species or doublet radicals, which Molpro can detect automatically. Molpro prints `SPIN SYMMETRY: Singlet` somewhere in the output file for my jobs. We could parse that instead, assuming it's similar for doublets and triplets.